src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Onramps style guide
+contentType: page
+template: basicDoc
+topics:
+  - onramps
+---
+
+import apmOnrampsIssuesExample from 'images/apm_screenshot-crop_onramps-issues-example.webp'
+
+import apmActiveEmptyState from 'images/apm_screenshot-crop_active-empty-state.webp'
+
```

Comment:
@ally-sassman It would be great to add a section in this style guide doc about the purposes of on-ramps content:
1. We want to enable users to get more value out of the UI
2. We want to help provide a larger context for why and how people can use these pieces of the UI to solve their problems and connect them to other parts of the UI
3. We want to provide info on how users can solve specific problems with the part of the UI they're looking at.

Reply:
Feel free to chat with me about this if you want to talk through it.

Reply:
**Notes**

The purpose is to help people with the thing they're looking at when it's not obvious.

The most important questions are the why and the how.

However, we want the UI to speak for itself, so we only use it when there's ambiguity. 

Users also use it to solve a problem (i.e., why is the Apdex page important to me?) 

There's a larger context - how does a piece of the UI fit into the larger context? I.e., how do APM and logs relate to each other? How does APM and infrastructure relate to either other? Here's how this aspect of APM relates to this piece of infrastructure. How are Issues related to Alerts? 

The secondary goal is if there's ambiguity/something's not clearly defined, we can take action as the docs team to provide a patchwork of sorts, filling in where there may be confusion.

The business goal is that in-product help will help users get more out of NR, and in turn we make more money.